### DISCUSSION TITLE
[DateRangePicker] choosing time issue

### DISCUSSION THREAD

**Discussion by abdullaev388** [rsuite/rsuite#3339] [Q&A]
I am using DateRangePicker with this format: "yyyy-MM-dd HH:mm" and also i set showOneCalendar prop to true.
When this props is true, there is no way to choose time for second date, only for first.

https://codesandbox.io/s/adoring-sea-wx6czj - You can open this link and try to choose with picker time for second date, you will not be able to.

**Comment by SevenOutman** [comment]
The single-calendar view of DateRangePicker is not suitable for specifying date+time. You may still need dual-calendar view.

**Reply by abdullaev388** [reply]
But single single-calendar more compact and our users find it better-looking. Is there any possibility that you add second button to specify second time?

**Reply by SevenOutman** [reply]
This would require design update, we need to discuss with our designer.

Where would you expect the set time widgets to be placed on the single-calendar view?

**Reply by abdullaev388** [reply]
a second time choosing button
<img width="254" alt="Screenshot 2023-08-24 at 11 55 03" src="https://github.com/rsuite/rsuite/assets/78722918/11461b10-e7fb-431b-8fa3-72b7bdf2f8d2">
